lib/doc_builder_testing/doc_builder_wrapper.rb

```diff
@@ -25,15 +25,16 @@ def default_builder_location
   end
 
   # @return [String] command to run builder for any platform
-  def run_build_command(script_file)
-    "#{@builder_exe} #{script_file} 2>&1"
+  def run_build_command(script_file, **arguments)
+    argument = arguments == {} ? nil : arguments.to_json.gsub('"', '\"').sub('{', '\{').sub('}', '\}')
```

Comment:
let's extract all this gsub logick to some separate method and let's write some unit tests specifically for those replacement

The gsubs like what is very easy to messed up even with minor change